tests/02_eradiate/01_unit/scenes/atmosphere/test_molecular_atmosphere.py

```diff
@@ -39,27 +39,22 @@ def test_molecular_atmosphere_afgl_1986(mode_ckd):
         mi_params.update(params.render(ctx))
 
 
-@pytest.fixture
-def ussa76_approx_test_absorption_data_set():
-    return data_store.fetch("tests/spectra/absorption/us76_u86_4-spectra-4000_25711.nc")
-
-
-def test_molecular_atmosphere_ussa_1976(
-    mode_mono, ussa76_approx_test_absorption_data_set
-):
+@pytest.mark.slow
+def test_molecular_atmosphere_ussa_1976(mode_mono):
     # ussa_1976() constructor produces a valid kernel dictionary
+    eval_w = [440.0, 550.0, 660.0] * ureg.nm
     atmosphere = MolecularAtmosphere.ussa_1976(
-        absorption_dataset=ussa76_approx_test_absorption_data_set,
+        wavelength_range=eval_w[
+            (0, -1),
+        ]
```

Comment:
If Black gets in your way, you can tell it to skip the line:
```suggestion
        ]  # fmt: skip
```